rust-toolchain

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2021-08-01"
+channel = "nightly-2021-11-05"
```

Comment:
Please keep the scope of a PR confined to the feature/fix it is implementing. If we want to change the nightly toolchain and update `Cargo.lock` then it should be done as a separate PR.

Reply:
Yeah, sure, will change it.

Will make PR for toolchain update from feature branch made previously here: [toolchain-and-workspace-update](https://github.com/aurora-is-near/aurora-engine/tree/toolchain-and-workspace-update)

Reply:
**Update:**
Here's the [PR from feature branch which contain just toolchain update](https://github.com/aurora-is-near/aurora-engine/pull/339)